src/shared/icons/svgs/outline-search2.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="10" cy="10" r="8" stroke="#F4F6F8" stroke-width="2"/>
+<path d="M22 22L18 18" stroke="#F4F6F8" stroke-width="2" stroke-miterlimit="10"/>
+</svg>
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M11 11L14 14" stroke="#F4F6F8" stroke-width="2"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 14C11.3137 14 14 11.3137 14 8C14 4.68629 11.3137 2 8 2C4.68629 2 2 4.68629 2 8C2 11.3137 4.68629 14 8 14ZM8 12C10.2091 12 12 10.2091 12 8C12 5.79086 10.2091 4 8 4C5.79086 4 4 5.79086 4 8C4 10.2091 5.79086 12 8 12Z" fill="#F4F6F8"/>
+</svg>
```

Comment:
Why does this contain 2 `svg` elements? This doen't seem right, we should only have one which is 16x16

It also shouldn't be named `outline-search2`, this variant is a glyph